### PR TITLE
feat: add header component

### PR DESCRIPTION
**Pull request by Cosmoboi-i**
Created header component with hover css.
Will add button functionality after the structure is approved.
image

### CODE REVIEW

**Comment by rohini-senthil** [comment]
There is also a header with no tabs. Can we have a boolean attribute to control this?

image

**Comment by rohini-senthil** [comment]
The logo needs to be in the corner by the way

File: src/components/header/header.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.header {
+  font-weight: 300;
+  height: 88px;
+  background: #051C2C;
+  color: white;
+
```
Comment (Chetan-MckinseyBEBR) [inline src/components/header/header.css:6]: Plz remove empty lines

File: src/components/header/header.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.nav-button {
+  color: white;
+  padding: 0 5px 15px 5px;
+}
+
+.content-padding {
+  height: 100%;
+  width: 63vw;
```
Comment (Chetan-MckinseyBEBR) [inline src/components/header/header.css:39]: reason for using vw?

Comment (Cosmoboi-i) [reply src/components/header/header.css:39]: The content region will always be a certain ratio of the viewport. We won't be using this in any element with width less that of the whole viewport. But if it's a convention to use percentage, I'll change it.

File: src/components/header/header.css (hunk 1 above)
Comment (Cosmoboi-i) [reply src/components/header/header.css:6]: 1. Added boolean to control navbar
2. removed empty lines
3. I am using gotskills website as a reference. It has the logo fairly in the middle. Have a look at this.
image
If the logo still needs to be in the corner, I'll do that.

**Review by Chetan-MckinseyBEBR — COMMENTED** [review]
- The css styles are not responsive to support tabs.
- See if u can use tailwind grids

File: src/components/header/__tests__/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Header from '..';
+
+describe('Header', () => {
+  it('should render correctly with a navbar when Navbar boolean is true', () => {
+    const { asFragment } = render(<Header {...{ hasNav: true }} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it('should render correctly without a navbar when Navbar boolean is false', () => {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/header/__tests__/index.test.jsx:10]: Add a new line

File: src/components/header/index.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import PropTypes from 'prop-types';
+
+export default function Header(props) {
+  return (
+    <div className="font-light h-[88px] text-white flex justify-center bg-[#051C2C]">
+      <div className="h-full xl:w-[63vw] lg:[70vw] md:[80vw] flex box-border">
+        <img src="./assets/McK_Logo.png" className="h-[70px] self-end pl-0 lg:pr-[180px] sm:pr-[90px] pt-0 pb-[5px]" />
+        {props.hasNav && (
+          <div className="grid grid-cols-3 gap-10 items-end">
+            <button className="text-[white] hover:text-[#00A9F4] hover:border-b-[5px] hover:border-b-[#00A9F4] hover:border-solid pt-0 pb-[15px] hover:pb-2.5 px-[5px]">
```
Comment (Chetan-MckinseyBEBR) [inline src/components/header/index.jsx:11]: create a central css color variable in assets rather than having the hex code in here

**Review by Chetan-MckinseyBEBR — CHANGES_REQUESTED** [review]
Add few comments

File: src/components/header/index.jsx (hunk 4 above)
Comment (Cosmoboi-i) [reply src/components/header/index.jsx]: Done

File: src/components/header/index.jsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Header(props) {
```
Comment (Chetan-MckinseyBEBR) [inline src/components/header/index.jsx:4]: destructure the props

**Review by Chetan-MckinseyBEBR — CHANGES_REQUESTED** [review]
Added one comment. Good to merge after this change

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Pull request merged**